app/invocation/invocation_details_card.tsx

```diff
@@ -137,7 +137,8 @@ export default class ArtifactsCardComponent extends React.Component<Props, State
                 <div className="invocation-section">
                   <div className="invocation-section-title">Actions</div>
                   <div>
```

Comment:
if buildMetrics is missing should we skip rendering? 0 might be misleading / incorrect

Reply:
yeah, you're right, done